source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,94 @@
+
+Deck section
+~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro_deck.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (deck section)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (deck section)
+
+===  ======================================================   ============================================================================================
+No.  Control                                                  Function
+===  ======================================================   ============================================================================================
+1    :hwlabel:`JOG WHEEL`                                     Scratches the track
+2    :hwlabel:`PLAY/PAUSE` button                             Plays/pauses a track in the respective Deck
+3    :hwlabel:`SYNC` button                                   Sync the tempo to other playing track
+4    :hwlabel:`SCRATCH` button                                Toggle the scratch mode
+5    :hwlabel:`TEMPO` slider                                  Adjust the track playing speed
+6    :hwlabel:`PITCH BEND` buttons -/+                        Lower (-) or Raise (+) the pitch temporarily
+7    :hwlabel:`REVERSE` button                                Hold for reverse play
+8    :hwlabel:`BASS`/:hwlabel:`MID`/:hwlabel:`TREBLE` knobs   Adjust the low/mid/high-frequency regions of the song
+9    :hwlabel:`MUTE` buttons                                  Mute low/mid/high-frequency regions
+===  ======================================================   ============================================================================================
+
+Mixer section
+~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro_mixer.svg
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (deck section)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (mixer section)
+
+===  ======================================================   ============================================================================================
+No.  Control                                                  Function
+===  ======================================================   ============================================================================================
+1    Channel faders                                           Adjust the output level for each channel
+2    Cross fader                                              Fades between left and right deck.
+3    :hwlabel:`A CUE` / :hwlabel:`B CUE` buttons              Sends the selected channel's audio to the headphone output
+===  ======================================================   ============================================================================================
+
+Known Issues
+------------
+
+Controls not yet working (or tested):
+
+- Library browsing
+- Cue control
```

Comment:
What do mean by that? The controller does not have hotcue pads (judging from photos I found on Google)

Reply:
Or are you referring to PFL?

Reply:
On 2nd thought: I compared the function of the Cue button on the controller with the Cue button in the Mixxx UI. And they seem to work similar (holding the Cue button plays the track from a fixed location... right?). So I guess the Cue button is actually working as it should be. It just didn't make sense to me... I'll update the documentation for that button as well.

Reply:
Done